packages/devtools_app/lib/src/ui/fake_flutter/fake_flutter.dart

```diff
@@ -1,35 +1,6 @@
-// Copyright 2018 The Chromium Authors. All rights reserved.
-// Use of this source code is governed by a BSD-style license that can be
-// found in the LICENSE file.
+// TODO(jacobr): rename this library to flutter with a followup CL.
 
-/// Collection of utility classes forked from package:flutter and dart:ui
-/// to make it easier to write UI code that can later be ported to flutter.
-///
-/// Once Hummingbird launches, this library should be removed completely.
-/// https://github.com/flutter/devtools/issues/69
-///
-/// Functionality such as I18N and shadows have been arbitrarily ripped out of
-/// these classes as it would be hard to implement writing directly to
-/// dart:html.
-library fake_flutter;
-
-import 'dart:async';
-import 'dart:collection';
-import 'dart:math' as math;
-
-import 'package:meta/meta.dart';
-
-import 'collections.dart';
-import 'dart_ui/dart_ui.dart' hide TextStyle;
-import 'dart_ui/dart_ui.dart' as ui;
-
-export 'collections.dart';
-export 'dart_ui/dart_ui.dart' hide TextStyle;
-
-part 'assertions.dart';
-part 'change_notifier.dart';
-part 'diagnosticable.dart';
-part 'foundation.dart';
-part 'print.dart';
-part 'text.dart';
-part 'text_span.dart';
+export '_real_flutter.dart'
+    if (dart.library.ui) '_real_flutter.dart'
+    if (dart.library.html) '_fake_flutter.dart'
+    if (dart.library.io) '_fake_flutter.dart' hide Element;
```

Comment:
Is this last line necessary anywhere?

Reply:
it will be if you run the dart:vm controller tests in bazel using the dart vm instead of flutter tests.